test/runtests.jl

```diff
@@ -64,9 +72,9 @@ cd(dirname(@__FILE__)) do
         error("Some tests exited with errors.")
     end
 
```

Comment:
I think I tracked it down via blame recently and it's been there for years. Probably safe to remove.